pallets/funding/src/lib.rs

```diff
@@ -1535,7 +1535,7 @@ pub mod pallet {
 				<T as Config>::SetPrices::set_prices();
 				instantiator::async_features::create_multiple_projects_at(inst, self.starting_projects.clone());
 
-				frame_system::Pallet::<T>::set_block_number(0u32.into());
+				// frame_system::Pallet::<T>::set_block_number(0u32.into());
```

Comment:
Should remove or uncomment

Reply:
Done, although it shouldn't have much effect since zombienet sets the block to 0 anyway, but felix had some problems without this line I think